services/booking-api/serverless.yml

```diff
@@ -0,0 +1,77 @@
+service: booking-api
+
+plugins:
+  - serverless-bundle
+  - serverless-add-api-key
+
+projectDir: ../../
+
+custom: ${file(../../serverless.common.yml):custom}
+
+package:
+  individually: true
+
+provider:
+  lambdaHashingVersion: 20201221
+  name: aws
+  runtime: nodejs12.x
+  stage: dev
+  region: eu-north-1
+  endpointType: regional
+  tracing:
+    lambda: true
+  apiGateway:
+    restApiId:
+      !ImportValue ${self:custom.stage}-ExtApiGatewayRestApiId
+    restApiRootResourceId:
+      !ImportValue ${self:custom.stage}-ExtApiGatewayRestApiRootResourceId
+    restApiResources:
+      booking:
+        !ImportValue ${self:custom.stage}-ExtApiGatewayResourceBooking
+
+  environment:
+    stage: ${self:custom.stage}
+    resourcesStage: ${self:custom.resourcesStage}
+
+  iam:
+    role:
+      statements:
+        - Effect: Allow
+          Action:
+            - ssm:GetParameter
+          Resource:
+            - !Sub arn:aws:ssm:${self:provider.region}:${AWS::AccountId}:parameter/bookingEnvs/${self:provider.stage}
+
+functions:
+  createBooking:
+    handler: lambdas/create.main
+    events:
+      - http:
+          path: booking
+          method: post
+          private: true
+          cors: true
+  cancelBooking:
```

Comment:
Would like to have one line break after every lambda declaration. :)

Reply:
Me too, but I followed the syntax in other serverless.yaml files. But if thats OK with line breaks, then I will add it.

Reply:
It's OK, create the linebreak. And if you see this elsewhere when you are in the code base, it's fine to correct (y)